GMC_identification.ipynb

```diff
@@ -0,0 +1,1603 @@
+{
```

Comment:
<p>Unlear what the central blob here is -- maybe you should re-plot the peak map with an aperture showing where the integration is taking place specifically</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>added marker as suggested.</p>